apps/portals/e2e/configs/bannersConfig.ts

```diff
@@ -0,0 +1,19 @@
+import { Portal } from './routesConfig'
+
+export type BannerType = 'beta' | 'cookies' | 'survey'
+
+type BannerConfig = Record<Portal, BannerType[]>
+const bannerConfig: BannerConfig = {
```

Comment:
Tracks full width alert banners and the order in which the banners should be closed, per Portal. For example, the NF portal displays  a survey and a cookies banner. Since the survey banner covers the cookies banner, the survey banner should be dismissed first.